admin_manual/configuration_files/encryption_configuration.rst

```diff
@@ -54,101 +62,134 @@ The encryption keys are stored in following folders:
-files".
+Encrypting External Mountpoints
+-------------------------------
 
-.. figure:: ../images/encryption4.png
+You and your users can encrypt individual external mountpoints. You must have 
+external storage enabled on your Admin page, and enabled for your users (see 
+:doc:`external_storage_configuration_gui`).
 
-Go to your Personal page and enter your password in the Encryption removal form, and your 
-files will all be decrypted.
+After setting up the external storage mountpoint, click the little gear icon at 
+the right to expose the encryption menu. Encryption is enabled by default.
 
-.. figure:: ../images/encryption5.png
+.. figure:: ../images/encryption13.png
 
-Your users will also have to follow this step to decrypt their files. If 
-something goes wrong with decryption, click the ``Restore Encryption Keys`` 
-button to re-encrypt your files, and then review your logfile to see what 
-happened. 
+To disable encryption, click the gear icon and un-check **encryption**.
 
-Enabling a File Recovery Key
-----------------------------
+Enabling Users' File Recovery Key
+---------------------------------
 
-If you lose your ownCloud password, then you lose access to your encrypted files. If one 
-of your users loses their ownCloud password their files are unrecoverable. You cannot 
-reset their password in the normal way; you'll see a yellow banner warning "Please provide 
-an admin recovery password, otherwise all user data will be lost".
+If you lose your ownCloud password, then you lose access to your encrypted 
+files. If one of your users loses their ownCloud password their files are 
+unrecoverable. You cannot reset their password in the normal way; you'll see a 
+yellow banner warning "Please provide an admin recovery password, otherwise all 
+user data will be lost".
 
 To avoid all this, create a Recovery Key. Go to the Encryption section of your 
 Admin page and set a recovery key password.
 
-.. figure:: ../images/encryption6.png
+.. figure:: ../images/encryption10.png
 
 Then your users have the option of enabling password recovery on their Personal 
 pages. If they do not do this, then the Recovery Key won't work for them.
 
 .. figure:: ../images/encryption7.png
 
-For users who have enabled password recovery, give them a new password and recover access 
-to their encrypted files by supplying the Recovery Key on the Users page.
+For users who have enabled password recovery, give them a new password and 
+recover access to their encrypted files by supplying the Recovery Key on the 
+Users page.
 
 .. figure:: ../images/encryption8.png
 
+You may change your Recovery Key password.
+
+.. figure:: ../images/encryption12.png
+
+Or disable it.
+
+.. figure:: ../images/encryption11.png
+
+Disabling Encryption
+--------------------
+
+You have the option of changing your mind and disabling the Encryption app. 
+Just click its Disable button on the Apps page.
```

Comment:
Admin settings > Server-side encryption > Uncheck checkbox is how to disable encryption now:
https://github.com/owncloud/documentation/issues/1053#issuecomment-95627319

But as per comment above, we strong advice against toggling that more then once